### PR TITLE
#526:replicated_firebase_logging_system

### PR DESCRIPTION
**Pull request by SanskarGubreley**
## Type of Change

- [ ] Bugfix
- [ x] New feature
- [ x] Enhancement
- [ x] Refactoring
- [ ] Dependency updates

## Description

Replicated the logging functions 

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [x ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

**Comment by ks-shree** [comment]
Please follow contribution guidelines in commit message and PR title.

File: Frontend/ui-common/src/Engineering/Helpers/Commons.purs [hunk 1]
```diff
@@ -25,7 +25,7 @@ import Data.Function.Uncurried (Fn2)
 import Data.Maybe (fromMaybe, Maybe(..))
 import Data.Number.Format (toStringWith, fixed) as Number
 import Effect (Effect)
-import Effect.Aff (Aff, makeAff, nonCanceler, try)
+import Effect.Aff (Aff, makeAff, nonCanceler, try, launchAff_)
```
Comment (ks-shree) [inline Frontend/ui-common/src/Engineering/Helpers/Commons.purs:28]: @SanskarGubreley Can you check imports if not used

**Comment by khuzema786** [comment]
Also, why is my commit being reflecting in your PR. Can you correct it to have only your 1 commit.

**Comment by ks-shree** [comment]
@SanskarGubreley Why your PR has all other changes ? Can you rebase and squash into one single commit

**Review by ks-shree — CHANGES_REQUESTED** [review]
Rebase and squash into single commit

**Pull request closed without merging**